inference-engine/thirdparty/clDNN/runtime/ocl/ocl_user_event.cpp

```diff
@@ -23,7 +23,7 @@ bool ocl_user_event::get_profiling_info_impl(std::list<cldnn::instrumentation::p
     }
 
     auto period = std::make_shared<instrumentation::profiling_period_basic>(_duration->value());
-    info.push_back({"duration", period });
+    info.push_back({ instrumentation::profiling_stage::executing, period });
```

Comment:
Did you intend to change "duration" into "executing"?

Reply:
Yeah, this change fixes "execution type" of ocl_user_event from CPU to GPU (since "duration" is used for CPU layers, and all profiling tools mark such kernels as CPU based and don't display kernel`s name)